ksp/src/main/kotlin/com/livefront/sealedenum/internal/ksp/KotlinPoetKsp.kt

```diff
@@ -21,7 +21,7 @@ import java.nio.charset.StandardCharsets.UTF_8
 internal fun KSClassDeclaration.toClassName(): ClassName {
     require(!isLocal()) { "Local/anonymous classes are not supported!" }
     val pkgName = packageName.asString()
-    println("$qualifiedName, $simpleName")
+    @Suppress("UnsafeCallOnNullableType")
```

Comment:
Huh, yeah I don't think of `qualifiedName` as nullable but that makes sense for anon/locals.  👍